File: drift/src/main/java/drift/com/drift/model/EndUser.java
package drift.com.drift.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by eoin on 28/07/2017.
 */

public class EndUser {

    @Expose
    @SerializedName("id")
    public Long id;

    @Expose
    @SerializedName("orgId")
    public Integer orgId;

    @Expose
    @SerializedName("name")
    public String name;

    @Expose
    @SerializedName("email")
    public String email;

    @Expose
    @SerializedName("avatarUrl")
    public String avatarUrl;

    @Expose
    @SerializedName("type")
    public String type;

}

File: drift/src/main/java/drift/com/drift/model/EndUser.kt
package drift.com.drift.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by eoin on 28/07/2017.
 */

class EndUser {

    @Expose
    @SerializedName("id")
    var id: Long? = null

    @Expose
    @SerializedName("orgId")
    var orgId: Int? = null

    @Expose
    @SerializedName("name")
    var name: String? = null

    @Expose
    @SerializedName("email")
    var email: String? = null

    @Expose
    @SerializedName("avatarUrl")
    var avatarUrl: String? = null

    @Expose
    @SerializedName("type")
    var type: String? = null

}
